code/game/gamemodes/meteor/meteors.dm

```diff
@@ -351,6 +353,20 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	dropamt = 1
 	threat = 100
 
+//Aurora bolt
+/obj/effect/meteor/aurora
+	name = "glowing bolt"
+	desc = "It's so pretty."
+	icon_state = "aurora"
+	hits = 0
+	hitpwr = 0
+	meteorsound = 'sound/effects/gravhit.ogg'
+	meteordrop = list(/obj/effect/temp_visual/impact_effect/ion)
+	dropamt = 1
+	
+/obj/effect/meteor/aurora/Initialize(mapload, target)
+	. = ..()
```

Comment:
```suggestion
	var/xxx = round(world.maxx/2)
	var/yyy = round(world.maxy/2)
	chase_target(locate(xxx, yyy, 2))
	. = ..()
```
This will make each bolt go towards the center of the station. It might not be the best but it does help more of them hit the windows.

I've also been asked to see if you would consider keeping the old event stuff too. Like the color changes to the starlight.